RecoEgamma/EgammaElectronProducers/python/electronPFIsolationValues_cff.py

```diff
@@ -72,21 +72,21 @@
 )
 
 elPFIsoValueCharged04 = elPFIsoValueCharged03.clone()
-elPFIsoValueCharged04.deposits[0].deltaR = cms.double(0.4)
+elPFIsoValueCharged04.deposits[0].deltaR = 0.4
```

Comment:
I was looking at a similar case earlier today, it looks like the following syntax works and can let us further simplify the replacement cases in vectors
```suggestion
elPFIsoValueCharged04 = elPFIsoValueCharged03.clone(deposits = {0: dict(deltaR = 0.4)} )
```